### ISSUE TITLE
Run migration in Docker container

### ISSUE THREAD

**Issue by rjw57** [uisautomation/django-boilerplate#4]
We may want to modify the "devserver" target to run migrate before starting the app. Something like the container we use in sms-deploy: https://github.com/uisautomation/sms-deploy/blob/master/webapp/Dockerfile#L56

**Comment by abrahammartin** [comment]
I don't like that to be on the Dockerfile as it is "not part of the app", it's more of a compose thing  (how you set up your database together with the app. Should it be then in docker-compose?

**Comment by rjw57** [comment]
It already is in docker-compose but I'd argue the migration is part of the app starting. Nonetheless, if it is contentious, we can re-visit it later. Closing.

**Comment by abrahammartin** [comment]
I didn't probably explain myself well enough. I meant to say, to execute the migration in the docker-compose task that runs the app before running the app. The reasonale is that is the docker-compose who creates and configure the DB and put it together with the app, and I though it makes more sense in there. You could run the container without a DB. If we include the migration in the dockerfile to run the container without failing you would have to have the db running already or override the entrypoint.
